cloudify_cli/commands/secrets.py

```diff
@@ -397,6 +399,112 @@ def provider_create(
     )
 
 
+@providers.command(
+    name='update',
+    short_help='Update a already exists secrets provider',
+)
+@cfy.options.secret_provider_name
```

Comment:
I think `.secret_provider_name` doesn't exist? :)

Anyway, I think it'd be better to have the name (in update and in delete) be an argument rather than a flag, ie. `@cfy.argument('secret-provider-name')`.
So that it's `cfy sec prov del provider1` rather than `cfy sec prov del --name provider1`